src/main/generated/resources/data/minestuck/tags/mob_effect/sopor_sickness_whitelist.json

```diff
@@ -0,0 +1,37 @@
+{
+  "values": [
+    "minecraft:speed",
+    "minecraft:slowness",
+    "minecraft:haste",
+    "minecraft:mining_fatigue",
+    "minecraft:strength",
+    "minecraft:instant_health",
+    "minecraft:instant_damage",
+    "minecraft:jump_boost",
+    "minecraft:nausea",
+    "minecraft:regeneration",
+    "minecraft:resistance",
+    "minecraft:fire_resistance",
+    "minecraft:water_breathing",
+    "minecraft:invisibility",
+    "minecraft:blindness",
+    "minecraft:night_vision",
+    "minecraft:hunger",
+    "minecraft:weakness",
+    "minecraft:poison",
+    "minecraft:wither",
+    "minecraft:health_boost",
+    "minecraft:absorption",
+    "minecraft:saturation",
+    "minecraft:glowing",
+    "minecraft:levitation",
+    "minecraft:luck",
+    "minecraft:unluck",
+    "minecraft:slow_falling",
+    "minecraft:conduit_power",
+    "minecraft:dolphins_grace",
+    "minecraft:bad_omen",
+    "minecraft:hero_of_the_village",
```

Comment:
Looking over this list, I don't think "hero of the village" should be blockable by sopor sickness. Maybe there are one or two others, but this one I feel pretty sure about.

Reply:
Very reasonable, I should have done a more extensive check. Similarly perhaps Bad Omen should be omitted